.github/workflows/publish.yml

```diff
@@ -53,11 +53,10 @@ jobs:
       actions: read
       contents: write
       id-token: write # Needed to access the workflow's OIDC identity.
-    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.10.0
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v2.0.0
     with:
       base64-subjects: "${{ needs.build.outputs.hashes }}"
       upload-assets: true
-      compile-generator: true # Workaround for https://github.com/slsa-framework/slsa-github-generator/issues/1163
```

Comment:
The above issue is closed and https://github.com/slsa-framework/slsa-github-generator/blob/main/internal/builders/generic/README.md#error-updating-to-tuf-remote-mirror-tuf-invalid-key mentions the workaround is only needed for 1.2.x.